### ISSUE TITLE
Support for FSR 2.1

### ISSUE THREAD

**Issue by NLMichel01** [fholger/vrperfkit#129]
Hi,
Just to thank you for creating the VR Performance Kit and to notify you of an update on AMD side: 
https://gpuopen.com/meet-fidelityfx-super-resolution-2-1/

I have no idea if the kit can support FSR 2.1, but would be amazing if it could of course! Thanks!

**Comment by armax9** [comment]
That would be great!

**Comment by krazysh01** [comment]
This has already been discused in issue #92 
It is not possible to add FSR2 to vrperfkit because FSR2.0 and 2.1 requires motion vectors which the majority of games do not provide

**Comment by d0x360** [comment]
> This has already been discused in issue #92 It is not possible to add FSR2 to vrperfkit because FSR2.0 and 2.1 requires motion vectors which the majority of games do not provide

True, but if the game supports DLSS2+ or XeSS then you can inject FSR2.x and presumably the upcoming FSR3.x when RDNA3 launches.

So at least it's something that will become more common since I imagine eventually all games will have 1 of the 3 which should mean FSR can take it's place.

**Comment by joaco05** [comment]
I think it is a good moment to mention that there are general fsr 3 mods for games using dlss3

**Comment by joaco05** [comment]
also is good to mention that i dunno if fsr3 would be a good idea because of the added input lag
